artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java

```diff
@@ -161,6 +161,11 @@
     */
    SimpleString HDR_ROUTING_TYPE = new SimpleString("_AMQ_ROUTING_TYPE");
 
+   /**
+    * The Routing Type for this message.  Ensures that this message is only routed to queues with matching routing type.
```

Comment:
The comment is relative to HDR_ROUTING_TYPE: `The Routing Type ...`

Reply:
Thanks, @brusdev. I just fixed it.